pkg/parser/runtime.go

```diff
@@ -133,8 +133,11 @@ func (n *Node) ProcessStatics(statics []types.ExtraField, event *types.Event) er
 		}
 
 		if value == "" {
-			clog.Debugf("Empty value for %s, skip.", printStaticTarget(static))
-			continue
+			//allow ParseDate to have empty input
+			if static.Method != "ParseDate" {
```

Comment:
Don't think if this is necessary because you don't ignore others statics with potential empty value (it was the case before). Just check with the team if there no weird behavior with that.

Reply:
My tests showed that this exception is mandatory. Node process just skip empty field...

Reply:
the second time I misunderstand the logic sorry. Forget it.